whois-commons/src/test/java/net/ripe/db/whois/common/rpki/DummyRpkiDataProvider.java

```diff
@@ -13,8 +13,14 @@
 @Profile({WhoisProfile.TEST})
 public class DummyRpkiDataProvider implements RpkiDataProvider {
 
+    private List<Roa> loadedRoas;
+
     @Override
     public List<Roa> loadRoas() {
+        if (loadedRoas!= null && !loadedRoas.isEmpty()){
+            return loadedRoas;
+        }
+
         try {
             return new ObjectMapper().readValue(getClass().getResourceAsStream("/rpki/roas.json"), Roas.class).getRoas();
```

Comment:
Rename `loadCustomRoas(final List<Roa> roas)` to `loadRoas(final List<Roa> roas)` to overload the existing method. Also call `loadRoas(final List<Roa> roas)` with the `ObjectMapper()...getRoas()` and put all logic into *one* method.